engine/lib/output.php

```diff
@@ -464,3 +464,40 @@ function _elgg_sane_validate_url($url) {
 	// Re-check now.
 	return filter_var($s, FILTER_VALIDATE_URL) ? $url : false;
 }
+
+/**
+ * Use to convert large positive numbers in to short form like 1K, 1M, 1B or 1T
+ * Example: elgg_number_format_short(7201); // Output: 7K
+ * Example: elgg_number_format_short(7201,1); // Output: 7.2K
+ *
+ * @param $n         input number to format
+ * @param $precision number of digits in decimal place (default = 0)
+ * @return string
+ *
+ * @since 3.1
+ */
+function elgg_number_format_short($n, $precision = 0) {
```

Comment:
almost a  i envisioned it, have a look at 
https://github.com/ColdTrick/entity_view_counter/blob/51f8164903e47667584bd94a47258e5169692c49/start.php#L102-L142

Reply:
don't copy the code ;) 

Thinks I think are better in our code
- language key for suffix
- less complexitiy in if statements `$n > 1000 ` is not noeeded